www/proxy-sso/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		proxy-sso
+PLUGIN_VERSION=		1.3
```

Comment:
if we replace the old one, can this be 2.0 instead? what do you think?

Reply:
ok.